### PR TITLE
Pie chart in popups

### PR DESCRIPTION
**Pull request by beatles1014225**
There is something wrong with data values. When the number of votes is greater than 1000, the value is understood as 1,000 instead of 1000. Therefore, some charts are as though only one value is plotted.

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -64,16 +65,16 @@ function reservesStyle(feature) {
...
-        fillOpacity : 0.6
+        fillOpacity : 0.7
     }
     );
     layer.bringToFront();
 }
 
 function zoomToFeature(feature){
-				map.fitBounds(feature.target.getBounds());
+				map.fitBounds(feature.target.getBounds(onEachFeature));
```
Comment (JanVan01) [inline js/data.js]: Is that needed?

File: js/stream_layers.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+
```
Comment (JanVan01) [inline js/stream_layers.js]: I think this is not used anywhere.

**Comment by JanVan01** [comment]
You added javascript files for d3 but get it from a cdn. I would remove the files.

File: js/data.js (hunk 1 above)
Comment (beatles1014225) [reply js/data.js]: No, I forgot to delete it

**Comment by JanVan01** [comment]
The problem with the data format is fixed by an update to the data on the server.

**Review by JanVan01 — APPROVED** [review]

**Pull request merged**